### PR TITLE
Fixed test resource checking in maven-checkstyle-plugin

### PR DESCRIPTION
**Pull request by Maxxx873**
- Made adjustments to the test resource checking in the maven-checkstyle-plugin
- Included SuppressWarningsHolder in the checkstyle.xml file
- Corrected the Java import layout in dev-build.xml to align with the ImportOrder "/^java./,javax,jakarta,org,com" specified in checkstyle.xml

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]
Thank you!

File: server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java [hunk 1]
```diff
@@ -293,7 +294,7 @@ void failedEventShouldBeStoredInDeadLetterUnderItsGroupId(RetryEventsListener re
 
         when()
             .get(EventDeadLettersRoutes.BASE_PATH + "/groups/" + GROUP_ID)
-        .then()
+            .then()
```
Comment (vttranlina) [inline server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java:297]: Not sure it is better for readable?
Usually unindent before `given`, `when`, `then`

Comment (Maxxx873) [reply server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java:297]: For me personally, in this case, indentation does not have a significant impact on readability. But, if it is necessary for the project and the team, I can try to fix it. It is advisable to add a rule for this case in checkstyle.xml. Should I do this?

Comment (vttranlina) [reply server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQEventDeadLettersIntegrationTest.java:297]: +1

File: server/data/data-cassandra/src/test/java/org/apache/james/vacation/cassandra/CassandraNotificationRegistryTest.java [hunk 2]
```diff
@@ -43,9 +43,11 @@ class CassandraNotificationRegistryTest implements NotificationRegistryContract
 
     @BeforeEach
     public void setUp(CassandraCluster cassandra) throws Exception {
-        notificationRegistry = new CassandraNotificationRegistry(zonedDateTimeProvider, new CassandraNotificationRegistryDAO(cassandra.getConf()));;
+        notificationRegistry = new CassandraNotificationRegistry(zonedDateTimeProvider, new CassandraNotificationRegistryDAO(cassandra.getConf()));
+        ;
```
Comment (chibenwa) [inline server/data/data-cassandra/src/test/java/org/apache/james/vacation/cassandra/CassandraNotificationRegistryTest.java:47]: THis trailing `;` is clearly unnecessary.

**Review by chibenwa — APPROVED** [review]
Thanks for the proposal!

Big :+1: on my side ;-)

**Review by chibenwa — APPROVED** [review]

**Pull request merged**